editor.planx.uk/package.json

```diff
@@ -15,7 +15,7 @@
     "@mui/material": "^5.15.10",
     "@mui/utils": "^5.15.11",
     "@opensystemslab/map": "1.0.0-alpha.3",
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#bea2192",
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#e6040d9",
```

Comment:
I'll update across projects once https://github.com/theopensystemslab/planx-core/pull/437 approved and merged to `main`